### ISSUE TITLE
Test a11y issue

### ISSUE THREAD

**Issue by lliskovoi** [cds-snc/notification-planning#1365]
[TEST ISSUE] ## Missing page title 

Describe what is failing
- pages are missing unique programmatic titles 
- include an image
- observed on pages x, y 

## Potential fix

- Use page structure in reverse breadcrumb format 
- What nuggets of information could help us fix the issue

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...
